### ISSUE TITLE
[HOLD for payment 2023-02-03] [$1000] Update New Dot Text Inputs

### ISSUE THREAD

**Issue by Luke9389** [Expensify/App#12445]
Tracking Issue: https://github.com/Expensify/App/issues/12251
Design Doc Section: https://docs.google.com/document/d/197fXHLlHM_RJA4-3MITwdu0N5e0HX5Q74sA4W80YMD8/edit#bookmark=id.9fht3irzbesn

### Update Text Inputs

We’re removing the box around our form elements and doing just a single underline instead. The bottom border will be increased to 2px, and we will remove the border radius and horizontal padding from the input as well. 
![Screen Shot 2022-11-03 at 12 24 22 PM](https://user-images.githubusercontent.com/49007721/199815486-57c6783c-3256-4a9a-98c3-b4c586169273.png)

### Update Dropdown Inputs

Same as above. 
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014d17d5f58b8a355f</li>
        <li>Upwork Job ID: 1600294214092935168</li>
    </ul>
</details>

**Comment by Luke9389** [comment]
@shawnborton, when you have a moment, could you provide a mockup for disabled dropdown pickers?
![Screen Shot 2022-11-03 at 1 05 23 PM](https://user-images.githubusercontent.com/49007721/199823654-fadafb57-aeaa-48b0-aac0-adbf5a5a9d3e.png)

I'm 99% sure I know what they'll look like (should just be a 1:1 with the color swaps, right?), but it'll be nice to have a mockup so WebQA can compare.

**Comment by shawnborton** [comment]
I suppose we could do a color swap, as that would be the most 1:1 thing to do here. I think if I had to pick, I would just make the input have 50% opacity (with no color changes from non-disabled state) so it would look something like this? 
<img width="349" alt="image" src="https://user-images.githubusercontent.com/2319350/199835084-33a23a0c-b96e-4c24-ba47-3ddbc91ee53d.png">

**Comment by Luke9389** [comment]
That looks nice. I'm a tad concerned about accessibility (any time we use opacity we end up getting non-contrasting colors), but we could take another pass through the styles later? Idk, what do you think? Is that a concern for you?

**Comment by shawnborton** [comment]
Maybe we just do a 1:1 color replacement for now, and then we can revisit this one later? That would be more in the spirit of how we're approaching this anyways. So that being said, I think we get this: 
<img width="1177" alt="image" src="https://user-images.githubusercontent.com/2319350/199846863-0c532767-6827-4bec-b85a-69727c6efe61.png">


Thoughts?

**Comment by Luke9389** [comment]
Yeah, I'm down to revisit this one. 

I like the look of the prior (opacity 50) version better, but this one does seem more accessible. Maybe we could add an icon or something to make it more obviously 'locked'. Perhaps an actual lock icon right here? Not sure if that helps from an accessibility standpoint... idk

![Screen Shot 2022-11-03 at 3 41 13 PM](https://user-images.githubusercontent.com/49007721/199848310-3d46f81a-2ec4-4a63-ac7c-c19590154fe0.png)

**Comment by shawnborton** [comment]
Yeah, those are all fair points but I would say let's table that as a separate project from just doing some color updates. 

In terms of the lock icon though, I can't say I've seen something like that before. This is what material design does FWIW:
<img width="815" alt="image" src="https://user-images.githubusercontent.com/2319350/199849459-8b15de23-0e73-4b2f-9345-08c21ff2b725.png">

**Comment by Luke9389** [comment]
Interesting. Maybe we can get away with the opacity 50 approach then. Seems like that's their strategy as well.

**Comment by grgia** [comment]
Not overdue, adding reviewing label

**Comment by shawnborton** [comment]
@grgia Is there a PR that we are currently reviewing for this? If so, could you please add me as a reviewer? Thanks!

**Comment by grgia** [comment]
done!

**Comment by Luke9389** [comment]
I just applied External to this to get a C+ reviewer for the linked PR above. 

I'm not sure why it started at $1000, I think that might be an error?

**Comment by parasharrajat** [comment]
> I'm not sure why it started at $1000, I think that might be an error?

Currently, the default value is 1000.

**Comment by kevinksullivan** [comment]
Sent offer to @parasharrajat

**Comment by parasharrajat** [comment]
PR is being reviewed. 


:ribbon: :eyes: :ribbon:  C+ reviewed

**Comment by Luke9389** [comment]
Wrapping this one up today.

**Comment by kevinksullivan** [comment]
@parasharrajat still waiting on your acceptance for  the job when you get a chance.

**Comment by Luke9389** [comment]
Update: The PR is shaping up nicely. I think it'd be reasonable to expect this to be merged by the end of next week. The current action item is @parasharrajat getting his dev environment back up so it can be C+ tested. In the meantime I'll be testing and providing screenshots for all 6 platforms.

**Comment by Luke9389** [comment]
updated to address all review comments

**Comment by Luke9389** [comment]
Still discussing on the PR

**Comment by Santhosh-Sellavel** [comment]
PR for this issue is causing a regression console warning.

It occurs on all platforms.
Easily reproducible on Android
Steps:
Run Android App
Just Reload Android App by pressing R from the metro terminal.
![Screenshot_1674765060](https://user-images.githubusercontent.com/85645967/214944026-c24414d3-3ed2-4c3c-85ab-3b2e4a2e4cc8.png)

**Comment by Santhosh-Sellavel** [comment]
@parasharrajat 
Please check the comment by @Prince-Mendiratta on the PR more details.

**Comment by Luke9389** [comment]
@kevinksullivan, would you be willing to make a reporting bonus Upwork job for @daraksha-dk, here? They [reported a problem](https://expensify.slack.com/archives/C049HHMV9SM/p1670434521851459) that was fixed in the PR linked to this one.

**Comment by Luke9389** [comment]
Oh, I see. Thanks for the heads up.

**Comment by kevinksullivan** [comment]
Hi @daraksha-dk , wasn't sure what your name was on upwork, but if you can apply here for reporting I'll send the offer!

https://www.upwork.com/jobs/~0117caf7067569db1a

**Comment by Luke9389** [comment]
@parasharrajat yep, that's here: https://github.com/Expensify/App/pull/14638

**Comment by kevinksullivan** [comment]
Invited @daraksha-dk . Let me know once you've accepted. Thanks!

**Comment by kevinksullivan** [comment]
All set here. Thanks everyone!

**Comment by Luke9389** [comment]
@0xmiroslav I'll look into that for you. You did provide some great help here.

**Comment by kevinksullivan** [comment]
@0xmiroslav just sent you an offer for this. Let me know when you accept!

https://www.upwork.com/jobs/~013478b85b34c26830

**Comment by MelvinBot** [comment]
📣 @parasharrajat! 📣

Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork.
Please follow these steps:

1. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
2. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
3. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.

![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/202326034-1a292e64-48fd-4fef-9f4f-7feaaeedfd9f.png)

Format:
```
Contributor details
Your Expensify account email: <REPLACE EMAIL HERE>
Upwork Profile Link: <REPLACE LINK HERE>
```

**Comment by MelvinBot** [comment]
✅ Contributor details stored successfully. Thank you for contributing to Expensify!

**Comment by kevinksullivan** [comment]
Weird, it had me send an offer again? Not sure what happened, but I am hoping once you "accept" once more I can take care of this

**Comment by kevinksullivan** [comment]
>https://www.upwork.com/jobs/~011b06f02351b42d01

This gives me an error @0xmiroslav 

<img width="551" alt="image" src="https://user-images.githubusercontent.com/22508290/221748975-dc2c31fe-9b45-48b3-9e14-f3814b598ac6.png">


so I don't think I'll be cancelling it anyways. Does it still show as active on your end?
